### ISSUE TITLE
new terms needed for Kids First

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#7143]
Hi Peter,
The new terms listed below are needed for Kids First curation. Can you let me know if these look okay and I can help with adding them.

| clinical term  from Kids First       | Parent HPO ID      |  Parent HPO Label                         | New term                                                                                                                             |
| ----------------------- | ----------- | --------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
| bifid renal artery      | HP\_0008776 | Abnormal renal artery morphology' | bifid renal artery                                                                                                                   |
| 3 cm. liver             | HP\_0410042 | Abnormal liver morphology'        | Decreased liver size                                                                                                                 |
| seminal cysts           | HP\_0000035 | Abnormal testis morphology'       | seminal vescile cyst: [https://radiopaedia.org/articles/seminal-vesicle-cyst-1?lang=us](https://radiopaedia.org/articles/seminal-vesicle-cyst-1?lang=us)   |
| complete tracheal rings | HP\_0002778 | Abnormal tracheal morphology'     | complete tracheal rings: [https://www.chop.edu/conditions-diseases/complete-tracheal-rings](https://www.chop.edu/conditions-diseases/complete-tracheal-rings) |
| Bilateral in-toeing     | HP\_0001288 | Gait disturbance'                 | intoeing                                                                                                                             |
| Thickened URETHRA       | HP\_0000795 | Abnormality of the urethra'       | Thickened urethra                                                                                                                    |

**Comment by pnrobinson** [comment]
@nicolevasilevsky never heard of bifid renal artery. Could this be accessory renal artery? I suspect that this is an incorrect name for something else, since I cannot find anything about it in PubMed.

**Comment by pnrobinson** [comment]
Liver hypoplasia also seems to be a really rare feature. In the literature, one finds Hypoplasia of specific liver lobes, but I could not find Hypoplasia of the entire liver. Do we have more information about this? Again I wonder if this was a name for something else?

**Comment by pnrobinson** [comment]
Intoeing is described here
https://www.ncbi.nlm.nih.gov/books/NBK499993/
This existing term is a specific type of intoeing https://hpo.jax.org/app/browse/term/HP:0001840
I will try to figure out whether we need to restructure. Intoeing is a a Gait disturbance.

**Comment by pnrobinson** [comment]
This term is fine

label Complete tracheal ring
Exact Synonym Complete tracheal rings (synonym type plural form)
Related Synonym Complete tracheal ring deformity 
Related Synonym CTRD ((synonym type abbreviation)
Definition: Continuous or nearly continuous configuration of the cartilaginous tracheal ring. Tracheal cartilage normally 

PARENT - Anomalous tracheal cartilage HP:0004468

PMID:31215789

Comment: The trachea is surrounded by 16-20 rings of hyaline cartilage; these rings are incomplete and horseshoe-shaped (C-shaped). In complete tracheal rings, the rings encircle the entire trachea (O-shaped) or nearly the entire trachea. 

Pulmonary artery sling is a vascular anomaly often seen in CTRD in which the left pulmonary artery (LPA) originates from the right pulmonary artery (RPA) and encircles the distal trachea and right primary bronchus before entering the hilum of the left lung. Pumonary artery sling should be coded separately (HP:0004961).

**Comment by pnrobinson** [comment]
With thickened Urethra, I wonder if they actually mean Urethral stricture (scarring leading to stenosis or the urethra)?

**Comment by nicolevasilevsky** [comment]
Thanks Peter! I'll pass these comments onto Deanne and I'll update some of the annotations with the terms you recommended.

I'll add Complete tracheal ring

I appreciate your help!

**Comment by pnrobinson** [comment]
@nicolevasilevsky Any word back on these terms? Otherwise I will add what I can this week.

**Comment by nicolevasilevsky** [comment]
I emailed Deanne to follow up.

**Comment by pnrobinson** [comment]
@nicolevasilevsky we are preparing the next release shortly. I think we have added all terms that are possible here. I will close this issue, and please reopen individual terms if Deanne can provide details that would address the above issues.
